diff --git a/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js b/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js
index db4bbcc..2f6560e 100644
--- a/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js
+++ b/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js
@@ -99,6 +99,8 @@ exports.networks = [
     { chainId: 333999, shortName: 'olympus' },
     { chainId: 421611, shortName: 'arb-rinkeby' },
     { chainId: 421613, shortName: 'arb-goerli' },
+    { chainId: 534351, shortName: 'scr-sepolia' },
+    { chainId: 534353, shortName: 'scr-alpha' },
     { chainId: 11155111, shortName: 'sep' },
     { chainId: 1313161554, shortName: 'aurora' },
     { chainId: 1313161555, shortName: 'aurora-testnet' },
